.gitignore

```diff
@@ -16,3 +16,4 @@
 coverage.txt
 codegpt
 dist
+.idea/
```

Comment:
It looks like this is just a patch to add a file or directory to the .gitignore file, so there isn't much to review in terms of code functionality. However, a couple of general points:

- It's always a good idea to double-check the changes you're making before committing them. Make sure you're adding the correct files/directories to the ignore list.
- If there are any sensitive files or directories in your project (e.g. containing passwords or API keys), make sure they're not accidentally added to the repository. It's a good idea to use a tool like git-secrets to scan for potential security issues.